test/e2e/multi-stage/e2e_test.go

```diff
@@ -121,12 +121,22 @@ func TestMultiStage(t *testing.T) {
 			success: true,
 			output:  []string{`run-as-script-success succeeded`},
 		},
+		{
+			name:    "e2e-claim",
+			args:    []string{"--unresolved-config=cluster-claim.yaml", "--target=e2e-claim"},
+			env:     []string{defaultJobSpec},
+			success: true,
+			output:  []string{"TODO"},
+		},
 	}
 
 	for _, testCase := range testCases {
 		testCase := testCase
 		framework.Run(t, testCase.name, func(t *framework.T, cmd *framework.CiOperatorCommand) {
 			cmd.AddArgs(testCase.args...)
+			if testCase.name == "e2e-claim" {
```

Comment:
No. Add this as a field to the test case or something - keying off of a string name is fragile.